.github/workflows/build.yml

```diff
@@ -285,11 +286,47 @@ jobs:
     name: Trigger Deployment
     needs: [build, lint, test]
     runs-on: ubuntu-latest
-    if: ${{ success() && github.ref == 'refs/heads/main' }}
     steps:
       - name: Trigger Deployment to QA and Staging
+        if: ${{ success() && github.ref == 'refs/heads/main' }}
         uses: benc-uk/workflow-dispatch@v1.1
         with:
           workflow: Deploy
           token: ${{ secrets.ACTIONS_API_ACCESS_TOKEN }}
           inputs: '{"qa": "true", "staging": "true", "research": "true", "sha": "${{ needs.build.outputs.IMAGE_TAG }}"}'
+
+      - name: Trigger Review App Deployment
```

Comment:
The other repos trigger the review app deployment early (before tests) so the deployment runs in parallel to the tests and make it faster for devs to see changes. We may want to do the same here

Reply:
Moved in to the build job